### PR TITLE
Faet/newtimeline

### PR DESCRIPTION
**Pull request by calilrenner**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import MyLikes from "./pages/MyLikes";
 import UserPosts from "./pages/UserPosts";
 
 export default function App() {
+  console.warn = () => {};
```
Comment (leoguzi) [inline src/App.js]: console.warn left behind;

File: src/pages/Timeline.js [hunk 2]
```diff
@@ -16,49 +17,54 @@ import {
...
+      } else {
+        return (
+          <>
+          {followedPosts.map((post) => <Post key={post.id} {...post} />)}
+          </>
+        )
+      }
+    }
+
+    console.log(followedPosts)
```
Comment (leoguzi) [inline src/pages/Timeline.js]: console.log left behind!

File: src/App.js (hunk 1 above)
Comment (calilrenner) [reply src/App.js]: Console warn removed.

File: src/pages/Timeline.js (hunk 2 above)
Comment (calilrenner) [reply src/pages/Timeline.js]: console log removed.

**Review by leoguzi — APPROVED** [review]

**Comment by sidneyouteiro** [comment]
Just fix the merge conflicts, then i will approve 👍🏽

**Comment by calilrenner** [comment]
Confliects fixed

**Review by sidneyouteiro — APPROVED** [review]

**Pull request merged**